tasks.py

```diff
@@ -592,6 +592,14 @@ def pylint(context, target=None, recursive=False):
         run_command(context, command)
 
 
+@task
+def docs_local(context):
```

Comment:
In general the invoke task names are verbs - so something like `serve_docs` (or even just `mkdocs`) would be a better name, IMO.